src/clarity-angular/emphasis/alert/alerts-pager.ts

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2017 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import {
+   Component,
+   EventEmitter,
+   Input,
+   OnDestroy,
+   OnInit,
+   Output
+} from "@angular/core";
+import {MultiAlertService} from "./providers/multi-alert-service";
+import {Alert} from "./alert";
+import {Subscription} from "rxjs/Subscription";
+
+
+@Component({
+   selector: "clr-alerts-pager",
+   templateUrl: "./alerts-pager.html",
+   host: {
+      "[class.alerts-pager]": "true"
+   }
+})
+export class AlertsPager implements OnInit, OnDestroy {
+   private subscription: Subscription;
```

Comment:
Can we give this a more descriptive name? Something like `multiAlertServiceChanged` or something?

Reply:
Yes, sounds reasonable.